rules/do-you-use-co-creation-patterns/rule.md

```diff
@@ -5,19 +5,20 @@ uri: do-you-use-co-creation-patterns
 authors:
   - title: Daniel Mackay
     url: https://ssw.com.au/people/daniel-mackay
-related: []
+related: 
+  - over-the-shoulder
 created: 2022-11-21T00:28:43.256Z
 guid: 4ff555fd-3f4c-416b-9fc3-1bf9409cc0ab
 ---
-These days pull requests are the de facto standard for getting code reviewed.  Once a developer has finished their change, they will typically submit a pull request and move on to their next task.  This allows for an asynchronous process to take place which may seem like a good idea, but this can also lead to inefficiencies.
+These days Pull Requests are the de facto standard for getting code reviewed.  Once a developer has finished their change, they will typically submit a Pull Request and move on to their next task.  This allows for an asynchronous process to take place which may seem like a good idea, but often is not and can also lead to inefficiencies.
```

Comment:
What's the reasoning behind capitalizing 'Pull Request'?  I notice that GitHub don't do this in their documentation.

For example: https://docs.github.com/en/pull-requests

Reply:
The thinking was that we usually refer to Pull Requests as a noun, in which case it should be capitalized.